banzai/main.py

```diff
@@ -233,6 +234,8 @@ def reduce_night():
                         help='ElasticSearch index to use for QC results')
     parser.add_argument('--es-doc-type', dest='elasticsearch_doc_type', default='qc',
                         help='Elasticsearch document type for QC records')
+    parser.add_argument('--bpm-not-required', dest='bpm_required', default=True, action='store_false',
+                        help='Do not require a bad pixel to be present to reduce data (Default BPM contains all zeros)')
```

Comment:
Might be more clear to have --bpm-required with a default of False. The double negative confuses me a bit, e.g that I have to set bpm-not-required to False in order to have banzai fetch a bpm. Or is this supposed to be --bpm-required? I am a bit confused.